src/router/routes.js

```diff
@@ -501,13 +501,17 @@ module.exports = [
 		}
 	},
 	{
-		path: '/lend/saved-search-beta',
+		path: '/lend/saved-search',
 		component: () => import('@/pages/Settings/SavedSearchBeta'),
 		meta: {
 			authenticationRequired: true,
 			excludeFromStaticSitemap: true,
 		}
 	},
+	{
+		path: '/lend/saved-search-beta',
```

Comment:
Do we need this fallback?

Reply:
eh prob not I guess it was only @tiarascoleman who may have memorized that one...

Reply:
I'll clean it up here and also the Prod version of the puppet configs.